MonoGame.Framework.Content.Pipeline/Audio/AudioContent.cs

```diff
@@ -284,7 +284,10 @@ public void ConvertFormat(ConversionFormat formatType, ConversionQuality quality
 
                 // Loop start and length in number of samples. Defaults to entire sound
                 loopStart = 0;
-                loopLength = data.Count / ((bitsPerSample / 8) * channelCount);
+                if (data != null && bitsPerSample > 0 && channelCount > 0)
+                    loopLength = data.Count / ((bitsPerSample / 8) * channelCount);
+                else
+                    loopLength = 0;
```

Comment:
This PR will get the audio content pipeline working for all inputs again.  Loop length will be set to zero, but the XAudio path is the only one that uses loopLength, and it would seem that a loopLength of zero doesn't adversely affect XAudio sound playback.

The audio content pipeline needs some rework.  The importers currently just copy the input file into a buffer. This input file is then used to generate the AudioFormat for the output (via ffprobe), which is wrong.  Simply using the output file in ffprobe doesn't work either because of the PCM format we convert to and ffprobe can't read it.

I also think that compressed input should be decompressed to PCM in the importer, and then PCM is converted to the output format in the processor.  So a few things that I think should be looked at further, but for now this will get people building audio content again.